### PR TITLE
Fix some problem related with publish version task

### PR DESCRIPTION
**Pull request by caiconghui**
This PR is mainly do following three things:
1. Add thread name in fe log to make trace problem more easy.
2. Add agent_task_resend_wait_time_ms config to escape sending duplicate agent task to be.
3. Skip to continue to update replica version when new version is lower than replica version in fe.

### CODE REVIEW

**Comment by caiconghui** [comment]
fix #4090

File: fe/src/main/java/org/apache/doris/catalog/Replica.java [hunk 1]
```diff
@@ -318,12 +318,17 @@ private void updateReplicaInfo(long newVersion, long newVersionHash,
...
-        LOG.debug("before update: {}", this.toString());
+        if (LOG.isDebugEnabled()) {
+            LOG.debug("before update: {}", this.toString());
+        }
 
         if (newVersion < this.version) {
-            // yiguolei: could not find any reason why new version less than this.version should run???
-            LOG.warn("replica {} on backend {}'s new version {} is lower than meta version {}",
-                    id, backendId, newVersion, this.version);
+            // This case means that replica meta version has been updated by ReportHandler before
```
Comment (yangzhg) [inline fe/src/main/java/org/apache/doris/catalog/Replica.java:326]: why the replica meta version been updated by ReportHandler before?
this is abnormal, I think using waring is better

Comment (caiconghui) [reply fe/src/main/java/org/apache/doris/catalog/Replica.java:326]: @yangzhg 
for example, now, the publish version daemon send publis verison task(2 3 4 5 6) to one be, and the be finish all publish version task, its replica version is 6 now, but fe need wait other be finish all publish version, now the replica version in fe may be 4, I think it is the common case that be replica version higher than fe replica version, when ReportHandler sync tablet. 
https://github.com/apache/incubator-doris/blob/14ac49dde557b54e9578d91f9dba7e148e5c975b/fe/src/main/java/org/apache/doris/master/ReportHandler.java#L473-L477

Comment (morningman) [reply fe/src/main/java/org/apache/doris/catalog/Replica.java:326]: Good explaination. I suggest to add this directly to the comment.

**Review by morningman — DISMISSED** [review]
LGTM

**Comment by morningman** [comment]
Hi @caiconghui , the FE code structure has been changed, please rebase the code to solve the conflicts

**Review by chaoyli — APPROVED** [review]

**Pull request merged**